.travis_e2e_test.sh

```diff
@@ -4,7 +4,7 @@ set -euxo pipefail
 
 eval "$(GIMME_GO_VERSION=1.10.2 gimme)"
 
-export BUILD_ID=build-891
+export BUILD_ID=build-3906
```

Comment:
Latest master build is 1035, so this doesn't make sense. If you want to test with a build from a branch/PR I can show you how to do that.